docs/topics/logging.rst

```diff
@@ -242,6 +242,48 @@ e.g. in the spider's ``__init__`` method::
 If you run this spider again then INFO messages from
 ``scrapy.spidermiddlewares.httperror`` logger will be gone.
 
+You can also filter messages by content using a substring or
+a regular expression. Create a ``logging.Filter`` subclass 
+and equip it with a regular expression pattern to
+filter out unwanted messages::
+
+    import logging
+    import re
+    
+    class ContentFilter(logging.Filter):
+        def filter(self, record):
+            m = re.search (r'.*\d\d\d [Ee]rror, retrying.+', record.msg)
+            if m.group(0) is not None:
+                return False
```

Comment:
It’s not really important, but I would refactor this for it to look a bit more clean, e.g.

```suggestion
            match = re.search(r'\d{3} [Ee]rror, retrying', record.message)
            if match:
                return False
```